modules/ROOT/pages/design-trusted-domains.adoc

```diff
@@ -0,0 +1,20 @@
+= Allowlisting Trusted Domains
```

Comment:
@failup We are missing a section that outlines domain inheritance and how that works for business units.